DR_0010_external-file-handling.md

```diff
@@ -13,7 +13,11 @@ The Cadasta Platform is inconsistent in its referencing of external static resou
 ## Suggested solution
 
 - Cadasta should adopt a standard practice of hosting all files locally on our servers.
-- These files should be located under `cadasta-platform/cadasta/core/static/` in their respective `css`, `fonts`, `img`, `js` folders.
+- New libraries or plug-ins should be located under `cadasta-platform/cadasta/core/static/`. 
+- A new folder with a descriptive name should be created.
+- A subfolder with the version number should be under the original folder.
```

Comment:
So we'd end up with a structure like `core/static/leaflet/1.0.4/{all the files here}`, right?